docs/INSTALL.md

```diff
@@ -458,6 +458,11 @@ the following:
   to provide a reliable delivery of notifications (e.g.
   `--zmqpubrawblock=tcp://127.0.0.1:28332` and
   `--zmqpubrawtx=tcp://127.0.0.1:28333`).
+- Make sure you do NOT use the bitcoind config setting `--rpcserialversion`.
+  Lightning depends on segwit transactions and the default rpc bitcoind
+  behaviour includes the witness data. Changes to this setting are safety
+  critical and might lead to loss of funds.
+  
```

Comment:
Maybe we can also link the discussion here?
https://github.com/bitcoin/bitcoin/issues/28730